packages/react-components/priority-overflow/src/types.ts

```diff
@@ -19,6 +19,15 @@ export interface OverflowItemEntry {
   groupId?: string;
 }
 
+export interface OverflowDivider {
```

Comment:
```suggestion
export interface OverflowDividerEntry {
```

nit: let's keep this consistent

Reply:
done